presto-spi/src/main/java/com/facebook/presto/spi/block/ByteArrayUtils.java

```diff
@@ -81,13 +53,13 @@ public static void gather( long[] source, int[] positions, int[] rowNumberMap, i
         targetOffset += ARRAY_BYTE_BASE_OFFSET;
         if (rowNumberMap == null) {
             for (int i = sourceOffset; i < end; i++) {
-                unsafe.putLong(target,  targetOffset, source[positions[i]]);
+                setLong(target,  targetOffset, source[positions[i]]);
```

Comment:
I wouldn't static import these calls as the coded loses readability with that as `setLong` sounds like a setter in this class, but it's not, etc.